src/main.js

```diff
@@ -237,34 +243,38 @@ async function buildComms(
   readBaseLines,
   locatordir,
 ) {
-  const myPeerID_s = await readBaseFile('private-id');
+  const myPeerIDS = await readBaseFile('private-id');
   const myPeerID = await promisify(PeerId.createFromJSON)(
-    JSON.parse(myPeerID_s),
+    JSON.parse(myPeerIDS),
   );
   const myPeerInfo = new PeerInfo(myPeerID);
   const ports = await readBaseLines('listen-ports');
-  for (const port of ports) {
+  ports.forEach(port => {
     if (port) {
       myPeerInfo.multiaddrs.add(port);
     }
-  }
+  });
 
   async function getAddressesForHostID(hostID) {
     const dirs = await fs.promises.readdir(locatordir);
     for (const d of dirs) {
       const idFile = path.join(locatordir, d, 'host-id');
       let f;
       try {
+        /* eslint-disable-next-line no-await-in-loop */
         f = await fs.promises.readFile(idFile, { encoding: 'utf-8' });
       } catch (ex) {
         if (ex.code !== 'ENOENT' && ex.code !== 'ENOTDIR') {
           throw ex;
         }
+        /* eslint-disable-next-line no-continue */
```

Comment:
weird that it doesn't like `continue` here, I'm not sure how else to achieve the goal of skipping the subsequent code on certain passes through the loop.

Reply:
Yeah, this pattern of usage doesn't seem to be supported by Airbnb at all, but I am at a loss for good alternatives to it.